cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc

```diff
@@ -16,25 +16,25 @@ The list of addressed issues are not cumulative; only the issues that are fixed
 |Description
 
```

Comment:
@Pubs-MV @divyabhushan are we getting these updates on the known issues article for the Compute Edition as well?

Reply:
@divyabhushan please confirm that we did.

Reply:
@grevach Yes, we added these updates under known issues for PCCE [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/767/files#diff-8ea760a8e7be938f27da286dd3f64a7461e10e7ba82ca876e3732a2ef336c32b)